pyvista/plotting/plotting.py

```diff
@@ -4132,7 +4143,7 @@ def show(self, title=None, window_size=None, interactive=True,
         #       See issues #135 and #186 for insight before editing the
         #       remainder of this function.
 
-        # Get camera position before closing
+        # Get camera position before closing.  This caches the camera position
         cpos = self.camera_position
```

Comment:
This doesn't actually cache anything.

And the docstring still mentions `cpos` as something that's returned.

Reply:
True.  I'm fixing this.